packages/jellyfish-address/src/constant/Regex.ts

```diff
@@ -0,0 +1,2 @@
+export const BASE_58 = new RegExp(/[1-9A-HJ-NP-Za-km-z]/)
+export const BECH_32 = new RegExp(/[0-9ac-hj-np-z]/)
```

Comment:
you could add length into your regex

Reply:
length check are done in other validator. Trying to make the logic should more or less guess the `type` when input half way.